pom.xml

```diff
@@ -170,7 +170,7 @@
 
         <!-- core lib dependency -->
         <grpc.version>1.53.0</grpc.version>
-        <netty.version>4.1.86.Final</netty.version>
+        <netty.version>4.1.94.Final</netty.version>
```

Comment:
What about 4.1.97. As it is used by grpc-java

https://github.com/grpc/grpc-java/commit/88b3484a602336b6688ca943d29b033c127dc533

Reply:
@lujiajing1126 We manage netty version separately AFAIK. We ignore the gRPC-Java dependency. The skywalking-eye license check verified the final use.